packages/material-ui-system/package.json

```diff
@@ -49,6 +49,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.4.4",
+    "@material-ui/styles": "^4.9.6",
```

Comment:
I don't think that we should have the styles as a dependency, as we try to progressively make it optional.

```suggestion
```